server/user_config.go

```diff
@@ -60,6 +60,7 @@ type UserConfig struct {
 	TFDownloadURL           string          `mapstructure:"tf-download-url"`
 	TFEHostname             string          `mapstructure:"tfe-hostname"`
 	TFEToken                string          `mapstructure:"tfe-token"`
+	TriggerWord             []string        `mapstructure:"trigger-word"`
```

Comment:
The mapstructure has to match the flag name, so this should be trigger-word**s**